### ISSUE TITLE
Compile failure when usign togerther with range/algorithm

### ISSUE THREAD

**Issue by RegexPuzzle** [boostorg/sort#35]
The following code fails to compile:

#include <boost/range/algorithm.hpp>
#include <boost/sort/sort.hpp>
void main(){}

**Comment by spreadsort** [comment]
Yes, that's due to a known scoping problem in boost/range/algorithm
